AdminUi/src/AdminUi/appsettings.override.json

```diff
@@ -38,7 +33,7 @@
         "SqlDatabase": {
           "Provider": "SqlServer",
           //"ConnectionString": "User ID=devices;Password=Passw0rd;Server=postgres;Port=5432;Database=enmeshed;" // postgres
```

Comment:
Shouldn't the placeholders for the postgres connection strings be changed also?
And on that note, did you test with postgres?

Reply:
> Shouldn't the placeholders for the postgres connection strings be changed also?
Good idea.

> And on that note, did you test with postgres?
I didn't, but I don't see a reason why it wouldn't work with postgres. I didn't change anything regarding data access. Did I miss something?